geocity/apps/permits/admin.py

```diff
@@ -1318,6 +1332,20 @@ def sortable_str(self, obj):
     sortable_str.short_description = _("3.2 Enquêtes public")
 
 
+class DuoConfigAdmin(admin.ModelAdmin):
```

Comment:
Add link to DUO security website explaining what is DUO for and how it can be configured